### ISSUE TITLE
datetimepicker is not keeping the old values

### ISSUE THREAD

**Issue by MickaelTH** [Laravel-Backpack/CRUD#1008]
Hi,

Small issue I think, the 'datetimepicker' fields is not keeping/showing the old values when validations rules for a form failed.

Ex, when the default 'defaultDate' is set in the 'datetime_picker_options' on the CRUD Controller.

![2017-09-25 002078](https://user-images.githubusercontent.com/26375917/30793596-45ca6cf6-a1ed-11e7-9f61-6bea4c16c642.png)

It will always show the default value and I think it is because the field isn't looking for old().

![2017-09-25 002077](https://user-images.githubusercontent.com/26375917/30793602-4cc23188-a1ed-11e7-93f9-49a3eeac534e.png)

Also this will happen in the EDIT view because it will not read the value from the crud object and display the default date instead.

**Comment by lloy0076** [comment]
I agree with your analysis.

**Comment by amipax** [comment]
Same problem here

**Comment by lloy0076** [comment]
@MickaelTH Is this still an issue for you?

**Comment by MickaelTH** [comment]
@lloy0076 yes it is still an issue

**Comment by tabacitu** [comment]
Hi @MickaelTH ,

The issue must have since been fixed - I just tested it and it works fine for me. A ```composer update``` should do it for you. The old value is taken into account over here:
![screen shot 2018-02-22 at 08 23 28](https://user-images.githubusercontent.com/1032474/36523333-b6545202-17a9-11e8-8f0b-5088a17ea9b2.png)

I'm going to close this, as [I've tested this on the current Backpack\CRUD and it works fine](https://cl.ly/1c2C28221G2a).

Cheers!
